tests/README.md

```diff
@@ -0,0 +1,72 @@
+# Test Data Automation
+Runs automted tests against Sentry demos on GCP, in order to generate errors and transactions to be sent to Sentry.io
+
+## Components / Moving parts
+- `create_job.sh` -> creates GCP cron job which hits Travis requests APIs to trigger build
+- `.travis.yml` -> runs automated tests / simulations
+- `conftest.py` -> Sauce Labs configuration (browsers) for frontend_tests
+- `backend_tests/backend_test.py` -> Hits /handled, /unhandled/, + /checkout backend demo APIs
+
+create_cron_job.sh -> GCP-cron job (runs every 20 min from midnight-6am) -> TravisCI (runs tests)
+
+# Tests
+
+## Setup
+python -m venv env
```

Comment:
good, this is critical or else it could pollute the global space on the VM, or even someone's virtual env that they're using for application monitoring.

We will keep using virtual environments as this is askin to using an individual node_modules directory and cuts down on the potential for SE's having errors due to this stuff. Localizes it.